app/serializers/book_suggestion_serializer.rb

```diff
@@ -0,0 +1,3 @@
+class BookSuggestionSerializer < ActiveModel::Serializer
+  attributes :author, :title, :link, :editor, :year, :price, :synopsis, :created_at, :updated_at
```

Comment:
In the index of booksuggestion in documentation you have user also

Reply:
Fixed